matchmaker-ui/src/app/pages/login-page/login-page.component.ts

```diff
@@ -13,22 +13,27 @@ import { LoginService } from 'src/app/shared/services/login-service/login.servic
   encapsulation: ViewEncapsulation.None
 
 })
-export class LoginPageComponent extends AppComponent implements OnInit {
```

Comment:
it was using the proxy config for the frontend code as well its going from localhost:4200 /login to localhost:8080/login.  Once i removed it that error went away.

Reply:
Ah ok. That's because I have some duplicate naming stuff that I'm working on. I think this is going to be fine as-is, and I can fix it in future builds.